ami/client/flowchart.py

```diff
@@ -230,9 +231,13 @@ def display(self, msg):
                 self.win.setCentralWidget(scrollarea)
 
             if msg.state and hasattr(self.widget, 'restoreState'):
+                self.widget.blockSignals(True)
```

Comment:
If we do not block signals on the widget we cause a checkpoint to be emitted when restoring the state, which is unnecessary.